src/main/java/careneighbors/guardian/Gift.java

```diff
@@ -0,0 +1,48 @@
+package careneighbors.guardian;
+
+
+import careneighbors.caregiver.Caregiver;
+import jakarta.persistence.*;
+
+@Entity
+public class Gift {
+
+    @Id
+    @GeneratedValue(strategy =  GenerationType.IDENTITY)
+    private Long id;
+
+    private String message;
+
+
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public Gift() {
+    }
```

Comment:
```suggestion
    protected Gift() {
    }
```
기본 생성자는 protected로 선언해요